src/mainScanner.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+import socket
+import subprocess
+import sys
+from datetime import datetime
+import json
+import os
+import threading
+import builtins
+from multi.scanner_thread import split_processing
+import logging
+from flask import Flask, render_template, request, redirect, url_for
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def homepage():
+    return render_template('index.html')
```

Comment:
For this to actually work and render the page, you need to add `index.html` inside your directory.
Consider copying the `src/static` and `src/templates` folder contents in this branch.